source/_patterns/02-molecules/branding/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Branding
+ */
+
+// Module dependencies
+import 'base';
+import 'atoms/image';
+import 'atoms/button';
```

Comment:
Does `branding` need button? I don't see a button atom used within it.

Reply:
It doesn't as-is, no. Had left in there as a part of the branding test you'd recommended originally, e.g., add a button to see if it works. I'll remove.